ids/datatables.py

```diff
@@ -184,16 +186,47 @@ def format(self, item):
         return FamilyCol.format(self, item)
 
 
+class IdsGlottocodeCol(Col):
+    __kw__ = {"bSortable": False}
+
+    def format(self, item):
+        if item.language.glottocode:
+            return external_link(
```

Comment:
I think for sites controlled by us - such as glottolog.org, we don't have to use "external_link".